README.md

```diff
@@ -27,22 +27,23 @@ http://help.thisdata.com
 
 #### Configuration
 
-Configure ThisData as follows:
+An example ThisData configuration is below. See `this_data/configuration.rb` for
+more options. For example, in production you will probably want asynchronous and
```

Comment:
I feel like this should say ".. in production we strongly recommend you enable asynchronous behaviour and disable debug logging" because the consequences could be bad if a user of the gem were not to turn on the asynchronous functionality and there is an issue with the API endpoint.